From 6b5bbdfaac7f216fe8a02c4cf29e5eb2aee5a409 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 15 Sep 2015 00:01:00 +0000
Subject: [PATCH] include sys/types.h for dev_t definition

Avoids the build failures when sys/types.h does not get included
indirectly through other headers.

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Submitted

 blktrace.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/blktrace.h b/blktrace.h
index 380aec7..944fc08 100644
--- a/blktrace.h
+++ b/blktrace.h
@@ -5,6 +5,7 @@
 #include <limits.h>
 #include <byteswap.h>
 #include <endian.h>
+#include <sys/types.h>
 
 #include "blktrace_api.h"
 #include "rbtree.h"
-- 
2.5.2

